### ISSUE TITLE
Add possibility to set view path for ViewLetter

### ISSUE THREAD

**Issue by notgosu** [rmrevin/yii2-postman#7]
Hi there!

It is common task to send emails from different modules but the same content(same view). For now i must copy-paste my view to all modules from which i send emails.

Please, add new property 'viewPath' to ViewLetter to handle this problem.

**Comment by rmrevin** [comment]
Fixed in version 2.2.3.
You can pass a parameter `view` as an alias.
```php
->setBodyFromView('@app/views/myemails/letter-text.php')
```
Or specify additional parameter `viewPath`.
```php
->setBodyFromView('letter-text', [], '@app/views/myemails')
```

**Comment by notgosu** [comment]
Great, thanks!

**Comment by tolik505** [comment]
Hi! In my case these methods don't work correctly. When I use ->setBodyFromView('@app/views/myemails/letter-text.php'), it build full path base/path/@app/views/..
Condition  if (strncmp($view, '@', 1)) {
            // example $view = '@app/view/email/letter-text.php'
            $path = \Yii::getAlias($view);
        } 
doesn't work.
When use ->setBodyFromView('letter-text', [], '@app/views/myemails'), it built such path 'letter-text'.
Please, can you check this method.

**Comment by rmrevin** [comment]
Show your `\Yii::$aliases`

**Comment by tolik505** [comment]
...
     '@yii' => '/var/www/largos/vendor/yiisoft/yii2'
    ]
    '@common' => '/var/www/largos/common'
    '@frontend' => '/var/www/largos/frontend'
    '@backend' => '/var/www/largos/backend'
    '@console' => '/var/www/largos/console'
    '@app' => '/var/www/largos/frontend'
    '@vendor' => '/var/www/largos/vendor'
    '@bower' => '/var/www/largos/vendor/bower'
    '@npm' => '/var/www/largos/vendor/npm'
    '@runtime' => '/var/www/largos/frontend/runtime'
    '@webroot' => '/var/www/largosfront'
    '@web' => ''
    '@creocoder' => [
        '@creocoder/translateable' =>
'/var/www/largos/vendor/creocoder/yii2-translateable/src'
    ]
...

2015-05-20 18:28 GMT+03:00 Revin Roman <notifications@github.com>:

**Comment by rmrevin** [comment]
Fixed in version 2.2.4
